lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -0,0 +1,156 @@
+"""
+Create DL3 FITS file from given data DL2 file,
+selection cuts and/or IRF FITS files.
+
+The selection cuts can be taken either from command-line arguments
+or a config file.
+
+Change the selection parameters as need be using the aliases.
+The default values are written in the DataSelection Component and
+in lstchain/data/data_selection_cuts.json
+
+Simple usage with argument aliases and default parameter selection values:
+
+lstchain_create_irf_files
```

Comment:
Hi! I think this is actually a "lstchain_create_dl3_file" instead of "lstchain_create_irf_files"

Reply:
Yes! Thank you for noticing it! Will correct it in the next commit.